packages/twenty-front/src/modules/object-record/record-table/components/RecordTableHeader.tsx

```diff
@@ -55,15 +55,16 @@ export const RecordTableHeader = ({
 }: {
   createRecord: () => void;
 }) => {
-  const { visibleTableColumnsSelector } = useRecordTableStates();
+  const { visibleTableColumnsSelector, hiddenTableColumnsSelector } =
+    useRecordTableStates();
 
   const scrollWrapper = useScrollWrapperScopedRef();
   const isTableWiderThanScreen =
     (scrollWrapper.current?.clientWidth ?? 0) <
     (scrollWrapper.current?.scrollWidth ?? 0);
 
   const visibleTableColumns = useRecoilValue(visibleTableColumnsSelector());
-  const hiddenTableColumns = useRecoilValue(visibleTableColumnsSelector());
+  const hiddenTableColumns = useRecoilValue(hiddenTableColumnsSelector());
```

Comment:
as I came across this piece of code it seemed wrong so I corrected it but I am less familiar with this part so not sure what if fixed, @charlesBochet

Reply:
+1